releaf-core/app/assets/stylesheets/releaf/include/error.scss

```diff
@@ -37,3 +34,8 @@ section.error a.return
     text-decoration: none;
 }
 
+body section.error:not(.dialog) > .body
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 3